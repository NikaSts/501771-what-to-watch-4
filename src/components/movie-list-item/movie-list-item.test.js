import React from 'react';
import renderer from 'react-test-renderer';
import MovieListItem from './movie-list-item.jsx';

const movie = {
  title: `Harry Potter`,
  image: `harry-potter`
};

it(`MovieCard should render correctly`, () => {
  const tree = renderer
    .create(
        <MovieListItem
          movie={movie}
          onMovieTitleClick={() => {}}
          onMovieCardMouseEnter={() => {}}
        />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
