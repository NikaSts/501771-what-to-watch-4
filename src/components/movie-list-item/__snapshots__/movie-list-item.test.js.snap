// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard should render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      muted={true}
      poster="img/harry-potter.jpg"
      preload="none"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Harry Potter
    </a>
  </h3>
</article>
`;
